### ISSUE TITLE
Ignore "extend schema" in .graphql files

### ISSUE THREAD

**Issue by gituserjava** [dimaMachina/graphql-eslint#1910]
We use below `extend` line in our .graphql file for federation and we get below error.

**Code snippet:**
````
extend schema @link(url: "https://specs.apollo.dev/federation/v2.3", import: ["@key"])

type Query {
     myQuery: String
}
````

**Error:**
`0:0 error Parsing error: [graphql-eslint] Error while loading schema: Directive
"@link" may not be used on SCHEMA`

Could you please advise on how this issue can be bypassed during graph ql eslint process?

**Comment by B2o5T** [comment]
this issue will be fixed after adding federation support https://github.com/B2o5T/graphql-eslint/issues/1824

**Comment by gituserjava** [comment]
Ok thank you @B2o5T, is there a tentative plan when this will be available?

**Comment by B2o5T** [comment]
Until end of year

**Comment by gituserjava** [comment]
Meanwhile is there a work around? I tried `# eslint-disable-next-line` and `# eslint-ignore` and both are not recognized by the library for .graphql files. Please advise.

**Comment by B2o5T** [comment]
They should works fine https://github.com/B2o5T/graphql-eslint/blob/master/packages/plugin/__tests__/eslint-directives.spec.ts

**Comment by gunes-sayin** [comment]
The directives work fine with files without parsing errors (extend in this case). But, they don't work with files that have extend on top. Is there a work around to ignore this part from parsing, until this feature is implemented?

**Comment by MarcCoquand** [comment]
Hi! Since this is reported as a parsing error, eslint directives are ignored. Is there any workaround?

**Comment by dimaMachina** [comment]
@MarcCoquand @gunes-sayin @gituserjava fixed in 4.3.0, your federation subgraphs should be parsed without parsing errors now https://github.com/dimaMachina/graphql-eslint/issues/2812#issuecomment-2517103848
